### DISCUSSION TITLE
Additional steps below for RHEL9

### DISCUSSION THREAD

**Discussion by olsonperrensen** [input-leap/input-leap#1657] [Ideas]
Starting from "Building on Linux" (CentOS 8+) : 

1. ` sudo subscription-manager repos --enable codeready-builder-for-rhel-9-x86_64-rpms`
2.  `sudo dnf install libXrandr-devel libXinerama-devel libICE-devel libSM-devel qt5-linguist`

Now you can continue the rest of the tutorial. Should finish with a `Build completed successfully` msg 

PS: You should additionally open ports to enable your server from being reached: 

`sudo firewall-cmd --zone=public --add-port=24800/tcp --permanent`

**Comment by shymega** [comment]
Hi,

Making this into a PR for this repo (https://github.com/input-leap/wiki-prs) would be welcome.
